### ISSUE TITLE
Failure to build v0.7.0

### ISSUE THREAD

**Issue by pnorman** [go-spatial/tegola#432]
6688122 fails to build with

```
pnorman@pippin:~/go/src/github.com/go-spatial/tegola/cmd/tegola$ GOPATH=$HOME/go go build
# github.com/go-spatial/tegola/cmd/tegola/cmd
cmd/server.go:47: undefined: shutdown
pnorman@pippin:~/go/src/github.com/go-spatial/tegola/cmd/tegola$ echo $?
2
```

**Comment by pnorman** [comment]
@ARolek I bisected the bug and it's the 6688122cc85da2296b68bf0f419447bd991cd120 commit that fails to build.

It looks like you squished the commits in #372 so I can't bisect further. I checked with the head of the PR, 28291c2a445a661c21d3a50736e3b6a5f95c4842, and that builds, so it's the rebasing that you did where the error came in, not in the code from @ear7h,

**Comment by ARolek** [comment]
@pnorman thanks for the report. I will get it fixed this week.

**Comment by ARolek** [comment]
@pnorman what version of Go are you using (`go version`)? I would suspect 1.7 or older based on this report since the shutdown command has a build flag of go 1.8 or newer. We're aiming to leverage the 3 most recent versions of Go, so 1.8, 1.9 and 1.10 as of right now.

**Comment by pnorman** [comment]
```
$ go version
go version go1.7.4 linux/amd64
```

This is the default on Debian Stretch, but I see there's some 1.8 packages. I'll need to test with them.

**Comment by pnorman** [comment]
`GOPATH=$HOME/go /usr/lib/go-1.10/bin/go build` builds. Is there a way to indicate a minimum required go version?

**Comment by ARolek** [comment]
@pnorman I will need to look into that. Right now the minimum version is documented in the [Building from source](https://github.com/go-spatial/tegola#building-from-source) section, but it would be ideal to indicate the version at build time to avoid this type of issue.

**Comment by ARolek** [comment]
I found a solution to this and just merged it into v0.7.0. Thanks for the report.
